frontend/src/components/StockData/StockData.jsx

```diff
@@ -46,7 +55,9 @@ export default function StockData(props) {
                     throw new Error("Stock data not found");
                 }
 
-                const stockSector = overviewData.Sector.charAt(0).toUpperCase() + overviewData.Sector.slice(1).toLowerCase()
+                const stockSector =
+                    overviewData.Sector.charAt(0).toUpperCase() +
+                    overviewData.Sector.slice(1).toLowerCase();
```

Comment:
This was already mentioned in a previous PR. You can write a util function `capitalizeFirstLetter` and invoke it from here.